src/query/v2/plan/operator.lcp

```diff
@@ -332,12 +335,13 @@ and false on every following Pull.")
    class OnceCursor : public Cursor {
     public:
      OnceCursor() {}
+     void PullMultiple(MultiFrame &, ExecutionContext &) override;
      bool Pull(Frame &, ExecutionContext &) override;
      void Shutdown() override;
      void Reset() override;
 
     private:
-     bool did_pull_{false};
+     size_t pull_count_{0};
```

Comment:
Why did you change this from a single bool to a counter?

Reply:
Unsure, I'll replace it